app/scripts/lib/account-tracker.js

```diff
@@ -110,25 +111,120 @@ export default class AccountTracker {
         ) {
           this._updateAccounts();
         }
-      }, this.onboardingController.store.getState()),
+      }, this.onboardingController.store.getState()), // is this right?
```

Comment:
I didn't change this, but it seems `this.onboardingController.store.getState()` maybe incorrect? I think it should be `this.preferencesController.store.getState())` since we are comparing to previous preferencesController state